packages/experimental/TabList/src/Tab/TabTokens.win32.ts

```diff
@@ -0,0 +1,153 @@
+import { buildUseTokens } from '@fluentui-react-native/framework';
+import type { Theme } from '@fluentui-react-native/framework';
+import { isHighContrast } from '@fluentui-react-native/theming-utils';
+import type { TokensFromTheme } from '@fluentui-react-native/use-styling';
+
+import type { TabTokens } from '..';
+
+export const tabStates: (keyof TabTokens)[] = [
```

Comment:
What of this file is different from the non-win32 version?

Reply:
The win32 token file uses different tokens for high contrast themes whereas the non-win32 tokens don't. I forked the HC support because I'm unable to test HC theming for mac.

Reply:
I see. In order to reduce duplication of code I think you should consider splitting the color tokens from the rest of the tokens (font/layout related) so that most of the tokens would be shared but the color ones can be platform specific. Button has an example of this.